cmake/std/atdm/README.md

```diff
@@ -254,6 +256,57 @@ $ srun ./checkin-test-sems.sh intel-opt-openmp \
   --local-do-all
 ```
 
+### chama/serrano
+
+Once logged on to `chama`  or `serrano`, one can
+directly configure and build on the login node (being careful not to overload
+the node).  But to run the tests, one must run on the compute nodes using the
+`srun` command.  For example, to configure, build and run the tests for say
+`MueuLu` on `serrano` or `chama`, (after cloning Trilinos on the `develop` branch) one
+would do:
+
+
+```
+$ cd <some_build_dir>/
+
+$ source $TRILINOS_DIR/cmake/std/atdm/load-env.sh intel-opt-openmp
+
+$ cmake \
+  -DTrilinos_CONFIGURE_OPTIONS_FILE:STRING=cmake/std/atdm/ATDMDevEnv.cmake \
+  -DTrilinos_ENABLE_TESTS=ON -DTrilinos_ENABLE_MueLu=ON \
+  $TRILINOS_DIR
+
+$ make -j16
+
+$ cat > job.sh << "EOF"
```

Comment:
@fryeguy52,

Is it necessary to create this temp `job.sh` script?    Is it possible to simply just run:

```
srun -N 1 --time=600 --account=<YOUR_WCID> ctest -j16
```

?